brave-snouts-frontend/src/styles/material/mixins/grids.mixin.scss

```diff
@@ -5,8 +5,7 @@
     virtual-scroller.full-items-grid {
         .items-grid {
             display: grid;
-            // grid-template-columns: repeat(4, minmax(0px, 1fr));
-            grid-template-columns: repeat(auto-fit, minmax(420px, 1fr));
+            grid-template-columns: repeat(4, minmax(0px, 1fr));
```

Comment:
Nope, leave it like before. 

This css applies to multiple places and works just fine. 1 thing you missed is that 
`user-items.component.html `

Must have `virtual-scroller `element with `full-items-grid` class and child with `items-grid` class. Then these will apply

`auto-fit` is here to scale depending on width size of the screen and not to be defined manually by us.

Reply:
You can see more at `grids.mixin.scss` L5-23